app/controllers/api/studio_assessments_controller.rb

```diff
@@ -0,0 +1,71 @@
+class Api::StudioAssessmentsController < ApplicationController
+    before_action :set_case_note, only: [:show, :update, :destroy]
+    respond_to :json
+
+    def show
+        render json: @studio_assessment
+    end
+
+    def create
+        @studio_assessment = authorize CaseNote.new(studio_assessment_params)
+        if @studio_assessment.save
+            render json: @studio_assessment, status: :created
+        else
+            render json: { error: 'Could not create studio assessment' }, status: :unprocessable_entity
+        end
+    end
+
+    def update
+        if @studio_assessment.update(studio_assessment_params)
+            render json: @studio_assessment, status: :ok
+        else
+            render json: { error: 'Could not update studio assessment' }, status: :unprocessable_entity
+        end
+    end
+
+    def destroy
+        if @studio_assessment.destroy
+            render json: @studio_assessment, status: :ok
+        else
+            render json: { error: 'Failed to delete studio assessment' }, status: :unprocessable_entity
+        end
+    end
+
+    def user_not_authorized
+        render json: { error: 'You are not authorized to perform this action' }, status: :unauthorized
+    end
+
+    private
+
+    def set_studio_assessment
```

Comment:
Divi and I talked about not doing authorization in set_studio_assessment and instead moving the authorization to each of the individual CRUD functions (so adding an authorize call to show, update, and destroy). I think that makes more sense? Would like your input! If we decide to do authorization in set_studio_assessments I can update the studio_asessment_policy pundit though. :)